src/Scan.js

```diff
@@ -302,13 +331,119 @@ class Scan extends React.Component {
 
     return checkIn.POST(requestData)
       .then(checkinResp => this.processResponse(checkinResp))
+      .then(checkinResp => this.processClaimReturned(checkinResp))
       .then(checkinResp => this.fetchRequests(checkinResp))
       .then(checkinResp => this.addScannedItem(checkinResp))
       .then(() => this.clearField('item.barcode'))
       .catch(resp => this.processError(resp))
       .finally(() => this.processCheckInDone());
   }
 
+  processClaimReturned(checkinResp) {
+    const fetchLoan = () => {
+      return checkinResp.loan;
+    };
+
+    const getAccounts = (loanId) => {
+      const {
+        mutator: {
+          accounts: { GET },
+        },
+      } = this.props;
+      const pathParts = [
+        'accounts?query=',
+        `loanId=="${loanId}"`,
+      ];
+      const path = pathParts.join('');
```

Comment:
I think the following would be just as clear: 
``` 
const path = `accounts?query=loanId=="${loanId}"`;
```

Reply:
Thanks for the observation, I already made the change.